core/src/main/java/cucumber/api/formatter/Formatter.java

```diff
@@ -4,11 +4,8 @@
 import cucumber.api.event.EventListener;
 
 /**
- * This is the interface you should implement if you want your own custom
- * formatter.
- *
- * @see EventListener
- * @see Plugin
+ * @deprecated as of version 4.0.0; use {@link Plugin } or {@link EventListener } instead.
```

Comment:
Is this "or" or "and"?

Reply:
Should be both, although sometimes ColorAware and/or StrictAware instead of Plugin.